requirements.txt

```diff
@@ -38,15 +38,13 @@ pygments==2.11.2
     # via -r requirements/requirements.in
 pymysql==1.0.2
     # via -r requirements/requirements.in
-pyodbc==4.0.32 ; platform_system != "Linux"
```

Comment:
this change surprises me - was this change intentional? It might be related to the test failures in CI (not sure)

How did you update these .txt files? I think `make py-requirements` is the way we usually do it (although just now I can't see that documented anywhere)

Reply:
cc @olsen232, any idea what happened here? Do you think it is responsible for the test fails in CI?

Reply:
> How did you update these .txt files? I think make py-requirements is the way we usually do it (although just now I can't see that documented anywhere)

Yes, `make-pyrequirements` was the way I did it too.

Reply:
I have added `pyodbc==4.0.32` back into the `requirements.txt`.

Reply:
Thanks. I've ticketed the issue at #582